### PR TITLE
FOGL-5809 Add documentation for SQLite plugin configuration options

### PR DESCRIPTION
**Pull request by MarkRiddoch**

### CODE REVIEW

File: docs/storage.rst [hunk 1]
```diff
@@ -84,6 +87,27 @@ readings will *not* cause the data in the previous storage system to be
...
+SQLite Plugin Configuration
+---------------------------
+
+The SQLite plugin has a more complex set of configuration options that can be used to configure how and when it creates more database to accommodate ore distinct assets. This plugin is designed to allow greater ingest rates for readings by separating the readings for each asset into a database table for that asset. It does however result in limiting the number of distinct assets that can be handled due to the requirement to handle large number of database files.
+
++-------------+
+| |sqlite_01| |
++-------------+
+
+  - **Purge Exclusions**: This option allows the user to specify that the purge process should not be applied to particular assets. ~The user can give a comma separated list of asset names that should be excluded from the purge process. Note, it is recommended that this option is only used for extremely low bandwidth, lookup data that would otherwise be completely purged from the system when the purge process runs.
```
Comment (ashish-jabble) [inline docs/storage.rst:99]: tilde (~) character

**Review by ashish-jabble — DISMISSED** [review]

**Comment by ashish-jabble** [comment]
just in case, given image screenshot  order of Purge exclusion at first position whereas expected in last position as per config order in https://github.com/fledge-iot/fledge/blob/develop/C/plugins/storage/sqlite/plugin.cpp#L80

File: docs/storage.rst (hunk 1 above)
Comment (MarkRiddoch) [reply docs/storage.rst]: Thanks, removed.

**Comment by MarkRiddoch** [comment]
> just in case, given image screenshot order of Purge exclusion at first position whereas expected in last position as per config order in https://github.com/fledge-iot/fledge/blob/develop/C/plugins/storage/sqlite/plugin.cpp#L80

Yes, not clear why that is happening. Will resolve in future as currently have an end user waiting for the functionality.

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**